spring-cloud-spanner-spring-data-r2dbc/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>spring-cloud-gcp</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.4-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
```

Comment:
check with Min on the version number (1.2.3 vs the current)

Reply:
I vote going with spring-cloud-gcp releases. It's easier on maintenance and makes more sense when we also have the upgraded version (4.x).

Reply:
makes sense. If needed we can leave a note @ `[cloud-spanner-r2dbc](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/tree/main/cloud-spanner-r2dbc)`